slackevents/inner_events.go

```diff
@@ -738,6 +961,52 @@ const (
 	TeamAccessRevoked = EventsAPIType("team_access_revoked")
 	// UserProfileChanged is sent if a user's profile information has changed.
 	UserProfileChanged = EventsAPIType("user_profile_changed")
+	// ChannelHistoryChanged The history of a channel changed
```

Comment:
I believe `AppRateLimited` is missing here

Reply:
The AppRateLimited event is already added in outer_events.go so wasnt able to replicate it here. I will remove this event from here.